code/modules/reagents/chemistry_properties/prop_positive.dm

```diff
@@ -492,7 +492,7 @@
 		var/obj/item/alien_embryo/embryo = content
 		if(embryo && istype(embryo))
 			if(embryo.counter > 0)
-				embryo.counter = embryo.counter - potency*1.5
+				embryo.counter = embryo.counter - potency*delta_time
```

Comment:
dont forget spacing . (( potency * delta_time)) instead of potency*delta_time

Reply:
Gotcha, saw a similar format to what I did. I'll correct it now.